gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/AttachmentHandlerService.java

```diff
@@ -74,4 +73,33 @@ public void storeAttachments(List<InboundMessage.Attachment> attachments, String
             }
         }
     }
+
+    public void storeEhrExtract(String fileName, String payload, String conversationId, String contentType) throws ValidationException {
+
+        if (fileName == null || fileName.isEmpty()) {
```

Comment:
You could use `StringUtils.isEmpty()` method in all of the if statements (it does exactly those 2 checks)